auslib/blobs/apprelease.py

```diff
@@ -115,8 +118,13 @@ def _getSpecificPatchXML(self, patchKey, patchType, patch, updateQuery, whitelis
         if isForbiddenUrl(url, updateQuery['product'], whitelistedDomains):
             return None
 
-        return '        <patch type="%s" URL="%s" hashFunction="%s" hashValue="%s" size="%s"/>' % \
+        patchXML = '        <patch type="%s" URL="%s" hashFunction="%s" hashValue="%s" size="%s"' % \
             (patchType, url, self["hashFunction"], patch["hashValue"], patch["filesize"])
+        additionalPatchAttributes = self._getAdditionalPatchAttributes(patch)
+        patchXML += additionalPatchAttributes['proof']
```

Comment:
Pretty close here - but you're still referencing "proof", which the base class has no idea about. See below for a bit more on what I had in mind.